services/web/server/src/simcore_service_webserver/users/_db.py

```diff
@@ -34,3 +39,22 @@ async def get_users_ids_in_group(conn: SAConnection, gid: GroupID) -> set[UserID
     async for entry in query_result:
         result.add(entry[0])
     return result
+
+
+async def list_user_permissions(
+    app: web.Application, *, user_id: UserID, product_name: str
+) -> list[Permission]:
+    engine = get_database_engine(app)
+
+    async with engine.acquire() as conn:
+        user_group_extra_properties = (
+            await GroupExtraPropertiesRepo.get_aggregated_properties_for_user(
+                conn, user_id=user_id, product_name=product_name
+            )
+        )
+    return [
+        Permission(
+            name="override_services_specifications",
+            allowed=user_group_extra_properties.override_services_specifications,
+        )
```

Comment:
so here we will have in the future other permissions, right?

Reply:
yes. my idea here would be to synchronise the user access roles from the backend with the ones of the frontend